src/avatar/clothes/Graphics.tsx

```diff
@@ -376,21 +376,48 @@ export class Bear extends React.Component<Props> {
   }
 }
 
+export class OnePlus extends React.Component<Props> {
```

Comment:
This should in its own component, not inside Graphics.tsx

Reply:
the other components are inside graphis.tsx